pkg/platform/defaults.go

```diff
@@ -169,8 +170,11 @@ func setPlatformDefaults(p *v1.IntegrationPlatform, verbose bool) error {
 			"-Dstyle.color=never",
 		}
 	}
-	if p.Status.Build.PersistentVolumeClaim == "" {
-		p.Status.Build.PersistentVolumeClaim = p.Name
+	//if p.Status.Build.PersistentVolumeClaim == "" {
```

Comment:
We can remove those comments to keep the source code more readable

Reply:
ok but I think that now with the map in place we have to have some additional logic to check if the key is present and then parsing the value. In this case `IsKanikoCacheEnabled()` helps because we don't need to do it everywhere.